crates/indexer/src/http/store_config.rs

```diff
@@ -93,12 +93,23 @@ pub async fn process(client: &Client, config_key: Pubkey, uri_str: String) -> Re
     client
         .db()
         .run(move |db| {
+            let subdomain_exists = select(exists(
+                store_config_jsons::table.filter(store_config_jsons::subdomain.eq(json.subdomain)),
+            ))
+            .get_result::<bool>(db);
+
+            if Ok(true) == subdomain_exists {
+                return Ok(())
+            }
+
             insert_into(store_config_jsons::table)
                 .values(&row)
                 .on_conflict(store_config_jsons::config_address)
                 .do_update()
```

Comment:
I went with a select then exit vs on_conflict because want 2 different on_conflict clauses. If the subdomain is the conflict then do nothing but if address then update but you can't have varying conditions.